sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -486,7 +519,7 @@ private static string[] parseActionErrorTarget(string targetReference)
             {
                 throw new InvalidOperationException("Expected an error with a target field referencing an action but did not get one");
             }
-            Regex _targetRegex = new Regex("#/tasks/(keyPhraseExtractionTasks|entityRecognitionPiiTasks|entityRecognitionTasks|entityLinkingTasks|sentimentAnalysisTasks)/(\\d+)", RegexOptions.Compiled, TimeSpan.FromSeconds(2));
+            Regex _targetRegex = new Regex("#/tasks/(keyPhraseExtractionTasks|entityRecognitionPiiTasks|entityRecognitionTasks|entityLinkingTasks|sentimentAnalysisTasks|extractiveSummarizationTasks)/(\\d+)", RegexOptions.Compiled, TimeSpan.FromSeconds(2));
```

Comment:
I would have totally missed this change if we didn't have the `AnalyzeOperationWithActionsError` test.

Reply:
I actually added the test because when the service added sentiment analysis task, I missed it. 
Glad before GA we added the test and that it is helpful :)